go/pserver/client/c/test/test_train.py

```diff
@@ -54,6 +54,9 @@ def main():
 
     # event_handler to print training and testing info
     def event_handler(event):
+        if isinstance(event, paddle.event.BeginPass):
+            master_client.paddle_start_get_records(event.pass_id)
```

Comment:
I think `paddle_start_get_records` is a implementation detail of cloud reader, we need to hide it inside the reader implementation.
The new pass is also indicated by `reader()` being called again (see [here](https://github.com/PaddlePaddle/Paddle/blob/develop/python/paddle/v2/trainer.py#L146)), so maybe it's better to put it [here](https://github.com/PaddlePaddle/Paddle/blob/develop/python/paddle/v2/reader/creator.py#L105) (in the `def reader` implementation), so every time a new pass begin, `reader()` is called, then `paddle_start_get_records` is called.